ui/components/Modal.js

```diff
@@ -0,0 +1,41 @@
+import { Modal, Backdrop, Fade } from '@mui/core'
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+
+const CustomModalWrapper = styled(Modal)(({ theme }) => ({
+   display : "flex",
+   alignItems : "center",
+   justifyContent : "center", 
+}))
+
+/**
+ *
+ * @param {{
+ *  open?: boolean,
+ *  Content?: JSX.Element,
+ *  handleClose?: (event: {}, reason: "backdropClick" | "escapeKeyDown") => void,
+ *  container?: React.ReactInstance | (() => React.ReactInstance)
+ * }} props
+ * @returns 
+ */
+
+export const CustomModal = ({
+    open, Content, handleClose, container
+  }) =>  { const theme = useTheme();
+    return (
+      <CustomModalWrapper
+        sx={{theme}}
+        open={open}
+        onClose={handleClose}
+        closeAfterTransition
+        BackdropComponent={Backdrop}
+        BackdropProps={{ timeout : 200, }}
+        container={container}
+      >
+        <Fade in={open} style={{ maxHeight : "90vh", overflow : "auto" }} >{Content}</Fade>
```

Comment:
we can use `sx` instead of `style` to keep it consistent.

Reply:
Cool.